### PR TITLE
Add reno release notes for 0.9 release

### LINKED ISSUE #2230: Adopt reno to write release notes incrementally
### What is the expected enhancement?

Right now when it's almost release time we have to go through the changelog file parse the intent of the small one line description that a patch author put there (which can often be difficult) and transform that into a human readable and friendly release note document. (see for example: https://github.com/Qiskit/qiskit/pull/211 or https://github.com/Qiskit/qiskit-terra/pull/1456). This manual process puts the burden of writing all the documentation on a single person or small group of people and puts pressure to get this done before the release.

Instead it would be better to do this incrementally as we add features, or make changes that warrant a release note. To do this we can leverage the reno project: https://docs.openstack.org/reno/latest/ this enables us to include a small yaml file with all the release documentation needed in a PR split up by sections. Then at release time reno will automatically assemble all those independent files into a single release notes document for a release. Using this means the burden for writing release notes is shared among all contributors to terra. When a patch author pushes a user facing change we should require that they write a detailed release note explaining the changes.

For an example of the output you can take a look at: https://docs.openstack.org/releasenotes/nova/stein.html

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

This commit starts the process of adding release notes using reno to
qiskit-terra. This notes for several of the user facing changes in the
0.9 release so far. We can continue to modify release notes after the fact
because they're tied to the git history for which release they're
associated with. New release notes can be added with: "reno new $name"
and you can generate rst output using "reno report". The plan is to
generate an rst file at each release and copy it into the documentation
repository as part of the release process (eventually this will hopefully
be automated).

Moving forward the expectation is that we'll require a PR to include
release documentation in the form of a reno release note before we merge
it, if it contains a user facing change. The goal being to switch the burden of
writing detailed release documentation off of solely me and share it among
the contributors to terra. Once we start using reno the expectation is that
PR authors will also write the release notes for any user facing change
they contribute.

Release notes differ from changelog in that they provide much more detail
to end users on what has changed, why it was changed, and how they can
adapt to it. Accordingly, the release notes are free form restructured text
that enables writing as much detail as needed to inform users of how to
adapt or leverage any changes in a new release. While the changelog is
just a record of a change that has potential end user impact and normally
doesn't provide the detail needed by users. Honestly the changelog
creation shouldn't be a manual process it should be automatically
generated using a combination of git and commit msg metadata.

We can continue to tweak and add on to these notes until we tag the release
and any additional notes files will be automatically included in the 0.9 notes.
After the release we can modify the release notes by modifying an existing
file.

### Details and comments

Fixes #2230

### CODE REVIEW

**Comment by mtreinish** [comment]
Just for people's reference the current rendered release notes (you can generate the rst locally by using `reno report` from the repo root) look like:

```
=============
Release Notes
=============

.. _Release Notes_0.8.0-266:

0.8.0-266
=========

.. _Release Notes_0.8.0-266_Prelude:

Prelude
-------

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

The 0.9 release includes many new features and many bug fixes. The biggest changes for this release are


.. _Release Notes_0.8.0-266_New Features:

New Features
------------

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new analysis pass ``CountOpsLongest`` was added to retrieve the number
  of operations on the longest path of the DAGCircuit. When used it will
  add a ``count_ops_longest_path`` key to the property set dictionary.
  You can add it to your a passmanager with something like::
  
      from qiskit.transpiler.passes import CountOpsLongestPath
      from qiskit.transpiler.passes import CxCancellation
      from qiskit.transpiler import PassManager
  
      pm = PassManager()
      pm.append(CountOpsLongestPath)
  
  and then access the longest path via the property set value with something
  like::
  
      pm.append(
          CxCancellation,
          condition=lambda property_set: property_set[
              'count_ops_longest_path'] < 5)
  
  which will set a condition on that pass based on the longest path.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- Two new functions, ``sech()`` and ``sech_deriv()`` were added to the pulse
  library module ``qiskit.pulse.pulse_lib`` for creating an unnormalized
  hyperbolic secant ``SamplePulse`` object and an unnormalized hyperbolic
  secant derviative ``SamplePulse`` object resepctively.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new kwarg option ``vertical_compression`` was added to the
  ``QuantumCircuit.draw()`` method and the
  ``qiskit.visualization.circuit_drawer()`` function. This option only works
  with the ``text`` backend. This option can be set to either ``high``,
  ``medium`` (the default), or ``low`` to adjust how much vertical space is
  used by the output visualization.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new kwarg boolean option ``idle_wires`` was added to the
  ``QuantumCircuit.draw()`` method and the
  ``qiskit.visualization.circuit_drawer()`` function. It works for all drawer
  backends. When ``idle_wires`` is set False in a drawer call the drawer will
  not draw any bits that do not have any circuit elements in the output
  quantum circuit visualization.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new PassManager visualizer function
  ``qiskit.visualization.pass_mamanger_drawer()`` was added. This function
  takes in a PassManager object and will generate a flow control diagram
  of all the passes run in the PassManager.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- When creating a PassManager you can now specify a callback function that
  if specified will be run after each pass is executed. This function gets
  passed a set of kwargs on each call with the state of the pass maanger after
  each pass execution. Currently these kwargs are:
  
   * pass_ (Pass): the pass being run
   * dag (DAGCircuit): the dag output of the pass
   * time (float): the time to execute the pass
   * property_set (PropertySet): the property set
   * count (int): the index for the pass execution
  
  However, it's worth noting that while these arguments are set for the 0.9
  release they expose the internals of the pass manager and are subject to
  change in future release.
  
  For example you can use this to create a callback function that will
  visualize the circuit output after each pass is executed::
  
      from qiskit.transpiler import PassManager
  
      def my_callback(**kwargs):
          print(kwargs['dag'])
  
      pm = PassManager(callback=my_callback)
  
  Additionally you can specify the callback function when using
  ``qiskit.compiler.transpile()``::
  
      from qiskit.compiler import transpile
  
      def my_callback(**kwargs):
          print(kwargs['pass'])
  
      transpile(circ, callback=my_callback)

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new kwarg ``log_level`` was added to ``qiskit.compiler.transpile()``.
  This can be used to change the default logging level for the Pass Manager
  used to perform the transpile. For example::
  
      import logging
      from qiskit.compiler import transpile
  
      transpile(circ, log_level=logging.DEBUG)
  
  will set the log level for the pass manager to DEBUG when transpiling the
  circuit circ.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new method ``filter()`` was added to the ``qiskit.pulse.Schedule`` class.
  This enables filtering the instructions in a schedule. For example,
  filtering by instruction type::
  
      from qiskit.pulse import Schedule
      from qiskit.pulse.commands import Acquire
      from qiskit.pulse.commands import AcquireInstruction
      from qiskit.pulse.commands import FrameChange
  
      sched = Schedule(name='MyExperiment')
      sched.insert(0, FrameChange(phase=-1.57)(device))
      sched.insert(60, Acquire(5))
      acquire_sched = sched.filter(instruction_types=[AcquireInstruction])

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- Additonal decomposition methods for several types of gates. These methods
  will use different decomposition techniques to break down a gate into
  a squence of CNOTs and single qubit gates. The following methods are
  added:
  
  +--------------------------------+---------------------------------------+
  | Method                         | Description                           |
  +================================+=======================================+
  | ``QuantumCircuit.iso()``       | Add an arbitrary isometry from m to n |
  |                                | qubits to a circuit. This allows for  |
  |                                | attaching arbitrary unitaries on n    |
  |                                | qubits (m=n) or to prepare any state  |
  |                                | of n qubits (m=0)                     |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.diagGate()``  | Add a diagonal gate to the circuit    |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.squ()``       | Decompose an arbitrary 2x2 unitary    |
  |                                | into three rotation gates and add to  |
  |                                |  a circuit                            |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.ucg()``       | Attach an uniformly controlled gate   |
  |                                | (also called a multiplexed gate) to a |
  |                                | circuit                               |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.ucx()``       | Attach a uniformly controlled (also   |
  |                                | called multiplexed) Rx rotation gate  |
  |                                | to a circuit                          |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.ucy()``       | Attach a uniformly controlled (also   |
  |                                | called multiplexed) Ry rotation gate  |
  |                                | to a circuit                          |
  +--------------------------------+---------------------------------------+
  | ``QuantumCircuit.ucz()``       | Attach a uniformly controlled (also   |
  |                                | called multiplexed) Rz rotation gate  |
  |                                | to a circuit                          |
  +--------------------------------+---------------------------------------+

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- Addition of Gray-Synth and Patel–Markov–Hayes algorithms for
  synthesis of CNOT-Phase and CNOT-only linear circuits.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new function ``random_circuit`` was added to the
  ``qiskit.circuit.random`` module. This function will generate a random
  circuit of a specified size by randomly selecting different gates and
  adding them to the circuit. For example, you can use this to generate a
  5 qubit circuit with a depth of 10 using::
  
      from qiskit.circuit.random import random_circuit
  
      circ = random_circuit(5, 10)

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new kwarg ``output_names`` was added to the
  ``qiskit.compiler.transpile()`` function. This kwarg takes in a string
  or a list of strings and uses those as the value of the circuit name for
  the output circuits that get returned by the ``transpile()`` call. For
  example::
  
      from qiskit.compiler import transpile
      my_circs = [circ_a, circ_b]
      tcirc_a, tcirc_b = transpile(my_circs,
                                   output_names=['Circuit A', 'Circuit B'])
  
  the ``name`` attribute on tcirc_a and tcirc_b will be ``'Circuit A'`` and
  ``'Circuit B'`` respectively.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- A new method ``equiv()`` was added to the ``qiskit.quantum_info.Operator``
  and ``qiskit.quantum_info.Statevector`` classes. These methods are used
  to check whether a second ``Operator`` object or ``Statevector`` is
  equivalent up to global phase.


.. _Release Notes_0.8.0-266_Upgrade Notes:

Upgrade Notes
-------------

.. releasenotes/notes/0.9/_util-removal-289fe623948983b8.yaml @ b'86ae899967612d04b60de3d617ee559bced59153'

- The previously deprecated ``qiskit._util`` module has been removed.
  ``qiskit.util`` should be used instead.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The ``QuantumCircuit.count_ops()`` method now returns an ``OrderedDict``
  object instead of a ``dict``. This should be compatible for most use cases
  since ``OrderedDict`` is a ``dict`` subclass. However type checks and
  other class checks might need to be updated.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The ``DAGCircuit.width()`` method now returns the total number quantum bits
  and classical bits. Before it would only return the number of quantum bits.
  If you require just the number of quantum bits you can use
  ``DAGCircuit.num_qubits()`` instead.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The function ``DAGCircuit.num_cbits()`` has been removed. Instead you can
  use ``DAGCircuit.num_clbits()``.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- Individual quantum bits and classical bits are no longer represented as
  tuples of registers and indexes. They are now instances of `Qubit` and
  `Clbit` classes. If you're dealing with individual bits make sure that
  you update checks to look for these new classes instead of tuples.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The default PassManager for ``qiskit.compiler.transpile()`` and
  ``qiskit.execute()`` has been changed to optimization level 1 pass manager
  defined at ``qiskit.transpile.preset_passmanagers.level1``. The previous
  default passmanager class
  ``qiskit.transpiler.preset_pass_managers.default.default_pass_manager`` has
  been removed.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The ``LegacySwap`` pass has been removed. If you were using it in a custom
  pass manager, it's usage can be replaced by the ``StochasticSwap`` pass,
  which is a faster more stable version. All the preset passmanagers have
  been updated to use ``StochasticSwap`` pass instead of the ``LegacySwap``.

.. releasenotes/notes/0.9/composite-gate-removal-f615a7d0d6a1214d.yaml @ b'86ae899967612d04b60de3d617ee559bced59153'

- The ``CompositeGate`` class has been removed. Instead you should
  directly create a instruction object from a circuit and append that to your
  circuit. For example, you can run something like::
  
      custom_gate_circ = qiskit.QuantumCircuit(2)
      custom_gate_circ.x(1)
      custom_gate_circ.h(0)
      custom_gate_circ.cx(0, 1)
      custom_gate = custom_gate_circ.to_instruction()

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated kwargs, ``seed`` and ``config`` for
  ``qiskit.compiler.assemble()`` have been removed use ``seed_simulator`` and
  ``run_config`` respectively instead.

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated converters
  ``qiskit.converters.qobj_to_circuits()`` and
  ``qiskit.converters.circuits_to_qobj()`` have been removed. Use
  ``qiskit.assembler.disassemble()`` and ``qiskit.compiler.assemble()``
  respectively instead.

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated kwarg ``seed_mapper`` for
  ``qiskit.compiler.transpile()`` has been removed. Instead you should use
  ``seed_transpiler``

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated kwargs ``seed``, ``seed_mapper``, ``config``,
  and ``circuits`` for the ``qiskit.execute()`` function have been removed.
  Use ``seed_simulator``, ``seed_transpiler``, ``run_config``, and
  ``experiments`` arguments respectively instead.

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated ``qiskit.tools.qcvv`` module has been removed
  use qiskit-ignis instead.

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated functions ``qiskit.transpiler.transpile()`` and
  ``qiskit.transpiler.transpile_dag()`` have been removed. Instead you should
  use ``qiskit.compiler.transpile``. If you were using ``transpile_dag()``
  this can be replaced by running::
  
      circ = qiskit.converters.dag_to_circuit(dag)
      out_circ = qiskit.compiler.transpile(circ)
      qiskit.converters.circuit_to_dag(out_circ)

.. releasenotes/notes/0.9/deprecated-compiler-dacafa4f0c1d0edd.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated function ``qiskit.compile()`` has been removed
  instead you should use ``qiskit.compiler.transpile()`` and
  ``qiskit.compiler.assemble()``.

.. releasenotes/notes/0.9/visualization-updates-30a8886dd3048a93.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The previously deprecated functions ``qiksit.visualization.plot_state`` and
  ``qiskit.visualization.iplot_state`` have been removed. Instead you should
  use the specific function for each plot type. You can refer to the
  following tables to map the deprecated functions to their equivalent new
  ones:
  
  ==================================  ========================
  Qiskit Terra 0.6                    Qiskit Terra 0.7+
  ==================================  ========================
  plot_state(rho)                     plot_state_city(rho)
  plot_state(rho, method='city')      plot_state_city(rho)
  plot_state(rho, method='paulivec')  plot_state_paulivec(rho)
  plot_state(rho, method='qsphere')   plot_state_qsphere(rho)
  plot_state(rho, method='bloch')     plot_bloch_multivector(rho)
  plot_state(rho, method='hinton')    plot_state_hinton(rho)
  ==================================  ========================

.. releasenotes/notes/0.9/visualization-updates-30a8886dd3048a93.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The ``pylatexenc`` and ``pillow`` dependencies for the ``latex`` and
  ``latex_source`` circuit drawer backends are no longer listed as
  requirements. If you are going to use the latex circuit drawers ensure
  you have both packages installed or use the setuptools extras to install
  it along with qiskit-terra::
  
      pip install qiskit-terra[visualization]


.. _Release Notes_0.8.0-266_Deprecation Notes:

Deprecation Notes
-----------------

.. releasenotes/notes/0.9/deprecate-U-CX-aef8e96ee489a141.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The ``U`` and ``CX`` gates have been deprecated. If you're using these gates
  in your code you should update them to use ``u3`` and ``cx`` instead. For
  example, if you're using the circuit gate functions ``circuit.u_base()``
  and ``circuit.cx_base()`` you should update these to be ``circuit.u3()`` and
  ``circuit.cx()`` respectively.

.. releasenotes/notes/0.9/deprecate-u0-gate-6581928ddddb45f1.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The ``u0`` gate has been deprecated in favor of using multiple ``iden``
  gates and it will be removed in the future. If you're using the ``u0`` gate
  in your circuit you should update your calls to use ``iden``. For example,
  f you were using ``circuit.u0(2)`` in your circuit before that should be
  updated to be::
  
      circuit.iden()
      circuit.iden()
  
  instead.

.. releasenotes/notes/0.9/qobj-as-dict-deprecate-9b0535234a3d605f.yaml @ b'876bb03be5a7f27f22fbf348f1da02367d06c938'

- The ``as_dict()`` method for the Qobj class has been deprecated and will
  be removed in the future. You should replace calls to it with ``to_dict()``
  instead.


.. _Release Notes_0.8.0-266_Bug Fixes:

Bug Fixes
---------

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- The definition of the ``CU3Gate`` has been changed to be equivalent to the
  canonical definition of a controlled ``U3Gate``.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- Support for n-qubit unitaries was added to the BasicAer simulator and
  ``unitary`` (arbitrary unitary gates) was added to the set of basis gates
  for the simulators


.. _Release Notes_0.8.0-266_Other Notes:

Other Notes
-----------

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- All the circuit drawer backends now willl express gate parameters in a
  circuit as common fractions of pi in the output visualization. If the value
  of a parameter can be expressed as a fraction of pi that will be used
  instead of the numeric equivalent.

.. releasenotes/notes/0.9/changes-on-upgrade-6fcd573269a8ebc5.yaml @ b'00fd7b2c431ac82d5a9ed0c010fc9cd50e421e7c'

- When using ``qiskit.assembler.assemble_schedules()`` if you do not provide
  the number of memory_slots to use the number will be infered based on the
  number of acquisitions in the input schedules.

.. releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml @ b'4b24d26f3662dd56f53434c70888d175d26e421f'

- Calls to ``PassManager.run()`` now will emit python logging messages at the
  INFO level for each pass execution. These messages will include the Pass
  name and the total execution time of the pass.
```

the version number 0.8.0-266 is based off of the git log (there are 266 commits to master since the 0.8.0 tag which is the last tag on the master branch). When we push a 0.9.0 tag that will automatically get reflected as the version number in the generated report (it will also increment the version for release notes committed to git after the 0.9.0 tag).

**Comment by mtreinish** [comment]
The latest commit adds documentation on using reno and writing release notes to the contributing doc. You can see it rendered here: https://github.com/Qiskit/qiskit-terra/blob/96f02a5524d35e20b617a2361d0a4fd7523f7590/CONTRIBUTING.md#release-notes

File: releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    of operations on the longest path of the DAGCircuit. When used it will
+    add a ``count_ops_longest_path`` key to the property set dictionary.
+    You can add it to your a passmanager with something like::
+
+        from qiskit.transpiler.passes import CountOpsLongestPath
+        from qiskit.transpiler.passes import CxCancellation
+        from qiskit.transpiler import PassManager
+
+        pm = PassManager()
+        pm.append(CountOpsLongestPath)
```
Comment (1ucian0) [inline releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml]: This is incorrect. It should be
```suggestion
        pm.append(CountOpsLongestPath())
```
Shall I change the file directly?

File: releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+    and then access the longest path via the property set value with something
+    like::
+
+        pm.append(
+            CxCancellation,
```
Comment (1ucian0) [inline releasenotes/notes/0.9/new-features-0.9-159645f977a139f7.yaml]: Same here:
```suggestion
            CxCancellation(),
```

**Comment by mtreinish** [comment]
Ok, I've applied the suggestions. Fwiw, we don't have to correct all the small issues in this review. We can do a lot of that in follow up PRs too. The whole point of reno is that it enables us to edit things as we build the release. It lets us switch to a more iterative method for building release documentation.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -177,6 +178,119 @@ The main rule to follow is:
...
+workflow for writing and compiling release notes.
+
+#### Adding a new release note
+
+Making a new release not is quite straightforward. Ensure that you have reno
+installed with::
+
+    pip install -U reno
+
+Once you have reno installed you can make a new release note by running in the
```
Comment (ajavadia) [inline CONTRIBUTING.md]:
```suggestion
Once you have reno installed you can make a new release note by running in
```

**Pull request merged**